jsonapi-resources.gemspec

```diff
@@ -4,12 +4,12 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'jsonapi/resources/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = 'jsonapi-resources'
+  spec.name          = 'sanger-jsonapi-resources'
   spec.version       = JSONAPI::Resources::VERSION
   spec.authors       = ['Dan Gebhardt', 'Larry Gebhardt']
   spec.email         = ['dan@cerebris.com', 'larry@cerebris.com']
   spec.summary       = 'Easily support JSON API in Rails.'
-  spec.description   = 'A resource-centric approach to implementing the controllers, routes, and serializers needed to support the JSON API spec.'
+  spec.description   = 'Forked from jsonapi-resources. A resource-centric approach to implementing the controllers, routes, and serializers needed to support the JSON API spec.'
   spec.homepage      = 'https://github.com/cerebris/jsonapi-resources'
```

Comment:
Homepage should be this repo.
Email should be empty or our rubygems email.
Authors could add yourself if you fancy 😄

Reply:
Thanks for pointing this. I've updated it to point to our team 😄 .